src/Control/Carrier.hs

```diff
@@ -1,4 +1,5 @@
 {-# LANGUAGE ConstraintKinds #-}
+-- | @since 1.0.0.0
 module Control.Carrier
```

Comment:
I feel like we should give a sentence of documentation or so, given that it is the top-level of so many other modules, but I’ll be damned if I can figure out the best way to approach it. Any ideas?

Reply:
Maybe: “Import this in modules defining effects or carriers.”

Reply:
Never mind, since it’s going away.